skins/cat17/templates/page_layouts/faq.html.twig

```diff
@@ -3,9 +3,12 @@
 {% block page_identifier %}faq{% endblock%}
 
 {% block main %}
-	<div id="faq"></div>
+	<div id="faq" content="{$ faq_content $}"></div>
```

Comment:
Let's try to keep the HTML valid by using the `data-` prefix for custom attributes. See https://stackoverflow.com/questions/24932391/are-custom-html-attributes-without-data-prefix-a-valid-attribute

Reply:
I actually named it `data-content` initially, and then on second thought I removed it because the words `data` and `content` sounded like synonyms in this context. 
Anyway, I didn't know it's considered invalid, so I'll change it. Thanks!